go-lang.md

```diff
@@ -42,7 +42,53 @@ Things we're not big fans of:
 
 >> All of our examples follow [a documented pattern using common tools](/wasm-languages/about-examples).
 
-TinyGo can compile to `wasm32-wasi`, and TinyGo apps can be [run on the Fermyon Platform](https://spin.fermyon.dev/go-components/).
+TinyGo can compile to `wasm32-wasi`, and TinyGo apps can be [run on the Fermyon Platform](https://spin.fermyon.dev/go-components/). You will need to [install TinyGo](https://tinygo.org/getting-started/) to do this example. Note that you also have to have [Go installed](https://go.dev/learn/)
+
+As with any Go project, the first step is to create a `go.mod` file:
+
+```
+module github.com/fermyon/example-go
+
+go 1.17
+```
+
+Next, create a simple Go program named `main.go`:
+
+```go
+package main
+
+func main() {
+	println("Content-Type: text/plain\n")
+    println("Hello, World!")
```

Comment:
There is some weird indentation here.

Do you think it makes sense to show an example that uses the Go SDK instead?

Reply:
I think I need to go rewrite the about_examples page to say that instead of being Wagi compatible, they are Spin compatible. I mean, I'm mainly using Spin as the executor now, so it doesn't seem necessary to stick with Wagi.